src/binder/binder_context.cpp

```diff
@@ -38,8 +38,10 @@ void BinderContext::AddRegularTable(const std::string db_name,
                                     const std::string table_alias,
                                     concurrency::TransactionContext *txn) {
   // using catalog object to retrieve meta-data
-  auto table_object = catalog::Catalog::GetInstance()->GetTableObject(
-      db_name, schema_name, table_name, txn);
+  auto table_object = catalog::Catalog::GetInstance()->GetTableObject(txn,
```

Comment:
We should probably rename this as `GetTableEntry` too.

Reply:
We should vote on this. I think `Get*CatalogEntry` would be best.

Reply:
I vote `Get*CatalogEntry`. I think the accuracy of function name is more important than length. And I think this name is not so long.

Reply:
I'd go with apavlo, ksaito7 and vote for ... Entry.

Reply:
Fine, I'll rename them to GetxxxCatalogEntry